### ISSUE TITLE
Updating Metadata post mint

### ISSUE THREAD

**Issue by thedonmon** [thuglabs/solana-nft-token-metadata-update#9]
Hello, wondering if this tool can be used to update metadata post mint? If so what is the point of updating the cachefile?

**Comment by jamland** [comment]
the purpose if update your NFTs after it was minted, like if you want to fix some typos in metadata.
you need reupload new metadata to arweave so you will have new updated data to use. 
you can't edit existing data on arweave but you can upload new data and link NFT to it

**Comment by jamland** [comment]
I will try to create step by step video tutorial in the next days since I had many questions about it already

**Comment by thedonmon** [comment]
Got it, I figured it was post mint, but wasnt sure why the candy machine needs to be updated, if you are just updating the mintaccount uri to point to the new data file?

**Comment by thedonmon** [comment]
> I will try to create step by step video tutorial in the next days since I had many questions about it already

Im a fool, I misread the instructions, the links just have to be in the correct format, not actually loading the whole cache file for the program to run... ok great thank you ser

**Comment by thedonmon** [comment]
> Otherwise newly updated links need to be formated to format of metaplex cache file. Put cache file into -> ./src/data/mainnet-beta-temp.json 

So if i upload everything to arweave myself without using metaplex upload, I just need to update the metadatauri links in a template cachefile?

**Comment by jamland** [comment]
@thedonmon yes, you can replace links with your own. Or you can simply use metaplex upload command on devnet. It will do the job for you and it will cost you nothing on devnet.

**Comment by darek292** [comment]
@jamland 

Can this also be done post "candy-machine withdraw" and after the NFT has been fully minted?

**Comment by jamland** [comment]
Yes, this script can be used only after mint is done.
Cache file is used to simplify. process of uploading new metadata.
After mint is done, you can create new candy machine with updated metadata on devnet for exmpl and use it with script on mainnet
